extra/foreman-debug-proxy

```diff
@@ -21,15 +25,18 @@ add_files /etc/xinetd.d/tftp
 [ -d "/var/lib/tftpboot" ] && add_cmd "find /var/lib/tftpboot -exec ls -ld {} +" "tftpboot_tree"
 
 # ISC bind
-add_files /etc/named.conf /var/log/named.log
-add_files /var/named/* /var/named/{data,dynamic,slaves}/*
+add_files /etc/named.conf
+add_files /var/log/named.log
+for SUBDIR in /var/named /var/named/{data,dynamic,slaves}; do
+  add_files $SUBDIR/
+done
 
 # Puppet master and CA
 add_files /var/lib/puppet/ssl/certs/$(hostname -f).pem /var/lib/puppet/ssl/certs/ca.pem
-add_files /etc/puppet/*.conf /etc/puppet/rack/* /etc/puppet/manifests/* /var/log/puppet/*.log*
+add_files /etc/puppet/*.conf /etc/puppet/rack/* /etc/puppet/manifests/*
```

Comment:
These are locations for older versions of puppet (pre 4.0). Current versions use ```/etc/puppetlabs/*```.